### ISSUE TITLE
Command pimcore:deployment:classes-rebuild not working anymore

### ISSUE THREAD

**Issue by mugge6** [pimcore/pimcore#4354]
# Bug Report

### Expected behavior

The following commands should work:
`./bin/console pimcore:deployment:classes-rebuild -c`
`./bin/console pimcore:deployment:classes-rebuild -d`

### Actual behavior

Both commands will fail with:
```
In ClassesRebuildCommand.php line 100:
                                                                      
Call to a member function cleanUpDeletedClassDefinitions() on null  
```
and 
```
In ClassesRebuildCommand.php line 82:
                                                                      
Call to a member function cleanUpDeletedClassDefinitions() on null  
```

### Steps to reproduce

Call the commands via CLI.

### Code example
Fix is to use get the ClassDefinitionManager (`\Pimcore::getContainer()->get(ClassDefinitionManager::class)`) from the container instead of the unset class variable (`$this->classDefinitionManager`).
```php
        if ($input->getOption('delete-classes')) {
            $questionResult = true;

            if ($input->isInteractive()) {
                $questionResult = $this->io->confirm(
                    '<error>You are going to delete classes that don\'t have class-definitions anymore. This could lead to data loss! Do you want to continue?</error>',
                    false
                );
            }

            if ($questionResult) {
                if ($output->isVerbose()) {
                    $output->writeln('---------------------');
                    $output->writeln('Delete Classes that don\'t have class-definitions anymore.');
                }

                foreach (\Pimcore::getContainer()->get(ClassDefinitionManager::class)->cleanUpDeletedClassDefinitions() as $deleted) {
                    if ($output->isVerbose()) {
                        [$class, $id, $action] = $deleted;
                        $output->writeln(sprintf('%s [%s] %s', $class, $id, $action));
                    }
                }
            }
        }

        $list = new ClassDefinition\Listing();
        $list->load();

        if ($output->isVerbose()) {
            $output->writeln('---------------------');
            $output->writeln('Saving all classes');
        }

        if ($input->getOption('create-classes')) {
            foreach (\Pimcore::getContainer()->get(ClassDefinitionManager::class)->createOrUpdateClassDefinitions() as $changes) {
                if ($output->isVerbose()) {
                    [$class, $id, $action] = $changes;
                    $output->writeln(sprintf('%s [%s] %s', $class, $id, $action));
                }
            }
        } else {
            foreach ($list->getClasses() as $class) {
                if ($class instanceof ClassDefinition) {
                    if ($output->isVerbose()) {
                        $output->writeln(sprintf('%s [%s] created', $class->getName(), $class->getId()));
                    }

                    $class->save(false);
                }
            }
        }
```

**Comment by brusch** [comment]
Already fixed cc117f07839630a73063109f424d0b9ecf6aa5bb

**Comment by dpfaffenbauer** [comment]
since that is quite an important feature, why don't release it with the next patch?
